concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-118	118	carry (generic)	700	CARRY	4
+Marrison-1967-909-119	119	carry (in hand)	412	CARRY IN HAND	4
+Marrison-1967-909-120	120	carry (on back)	465	CARRY ON BACK	2
+Marrison-1967-909-121	121	cat	1208	CAT	2
+Marrison-1967-909-122	122	catch	702	CATCH	2
+Marrison-1967-909-123	123	caterpillar	2247	CATERPILLAR	2
+Marrison-1967-909-124	124	cattle	1153	CATTLE	2
+Marrison-1967-909-125	125	cave	1155	CAVE	2
+Marrison-1967-909-126	126	centipede	1974	CENTIPEDE	2
+Marrison-1967-909-127	127	chaff	2363	CHAFF (HUSK)	4
+Marrison-1967-909-128	128	chair	1476	CHAIR	2
+Marrison-1967-909-129	129	charcoal	1188	CHARCOAL	2
+Marrison-1967-909-130	130	cheap	1887	CHEAP	2
+Marrison-1967-909-131	131	cheek	1730	CHEEK	2
+Marrison-1967-909-132	132	chest	1592	CHEST	2
+Marrison-1967-909-133	133	chicken	1318	CHICKEN	2
+Marrison-1967-909-134	134	chief	301	CHIEFTAIN	2
+Marrison-1967-909-135	135	child	2099	CHILD	2
+Marrison-1967-909-136	136	chillie			
+Marrison-1967-909-137	137	chin	1510	CHIN	2
+Marrison-1967-909-138	138	choose	1750	CHOOSE	2
+Marrison-1967-909-139	139	claw	72	CLAW	2
+Marrison-1967-909-140	140	clay	1399	CLAY	2
+Marrison-1967-909-141	141	clean	704	CLEAN	2
+Marrison-1967-909-142	142	climb	1102	CLIMB	2
+Marrison-1967-909-143	143	clock	1323	CLOCK	2
+Marrison-1967-909-144	144	cloth	1616	CLOTH	2
+Marrison-1967-909-145	145	clothes	1895	CLOTHES	2
+Marrison-1967-909-146	146	cloud	1489	CLOUD	2
+Marrison-1967-909-147	147	coat	603	COAT	2
+Marrison-1967-909-148	148	cock	1511	ROOSTER	2
+Marrison-1967-909-149	149	cold	1287	COLD	2
+Marrison-1967-909-150	150	comb	1475	COMB	2
+Marrison-1967-909-151	151	come	1446	COME	2
+Marrison-1967-909-152	152	cook	1100	COOK (SOMETHING)	2
+Marrison-1967-909-153	153	cook3			
+Marrison-1967-909-154	154	copper	1825	COPPER	2
+Marrison-1967-909-155	155	cotton	1850	COTTON	2
+Marrison-1967-909-156	156	cough	879	COUGH	2
+Marrison-1967-909-157	157	count	1420	COUNT	2
+Marrison-1967-909-158	158	country	1300	COUNTRY	2
+Marrison-1967-909-159	159	cover	761	COVER	2
+Marrison-1967-909-160	160	cow	1007	COW	2
+Marrison-1967-909-161	161	cowrie			
+Marrison-1967-909-162	162	crab	2348	CRAB	2
+Marrison-1967-909-163	163	creeper			
+Marrison-1967-909-164	164	crooked	297	CROOKED	2
+Marrison-1967-909-165	165	crow (bird)	913	CROW	4
+Marrison-1967-909-166	166	cruel	500	CRUEL	2
+Marrison-1967-909-167	167	cry (weep)	1839	CRY	2
+Marrison-1967-909-168	168	cucumber	496	CUCUMBER	2
+Marrison-1967-909-169	169	cup	476	CUP	2
+Marrison-1967-909-170	170	curry			
+Marrison-1967-909-171	171	curse	8	CURSE	2
+Marrison-1967-909-172	172	cut (generic)	1432	CUT	4
+Marrison-1967-909-173	173	cut (with axe)	3151	CUT (WITH AXE)	4
+Marrison-1967-909-174	174	cut (with knife)	2156	CUT (WITH KNIFE)	2
+Marrison-1967-909-175	175	dance	1879	DANCE	2
+Marrison-1967-909-176	176	dao			
+Marrison-1967-909-177	177	dark	706	DARK	2
+Marrison-1967-909-178	178	daughter	1357	DAUGHTER	2
+Marrison-1967-909-179	179	dawn	729	DAWN	2
+Marrison-1967-909-180	180	day	1260	DAY (24 HOURS)	2
```

Comment:
more likely day (not night): http://concepticon.clld.org/parameters/1225 This is the expected default. We should think of adding a relation (DAY -> DAY (OT NIGHT), DAY -> DAY (24 hours)). But default is mostly DAY (NOT NIGHT), as the list also has "night"